src/modules/User/About.js

```diff
@@ -71,7 +72,8 @@ class About extends Component {
                        <div className="userpage__section--about__headingContainer">
                          <div>
                            <h4 className="userpage__section--about__heading">About</h4>
-                           <Edit handleClick={this.startEditing} conditionalEnable classModifier="horizontalMargin" />
+                           <FontAwesomeIcon icon="edit" onClick={this.startEditing} className="ml_05" focusable />
+                           {/* <Edit handleClick={this.startEditing} conditionalEnable classModifier="horizontalMargin" /> */}
```

Comment:
Why is this commented out?

Reply:
This particular comment could be deleted, but at other places I have left commented code, so that it would be helpful when styling svgs.

Reply:
I don't understand. I thought there were no more SVG styling.

Reply:
We will need to style the icons (svgs), for example: color, size, hover properties.